### ISSUE TITLE
Some related work

### ISSUE THREAD

**Issue by DYR1** [git-disl/awesome_LLM-harmful-fine-tuning-papers#1]
We recently proposed a MoGU framework that can be used as a post-alignment strategy to improve the security of LLMs, and this work has been accepted by NIPS 2024. ([MoGU](https://arxiv.org/abs/2405.14488))

Besides, we recently released "[Towards Secure Tuning: Mitigating Security Risks Arising from Benign Instruction Fine-Tuning](https://arxiv.org/abs/2405.14488)", which belongs to protecting the security of LLM during fine-tuning.

I believe that adding these works to your list will be of some help to the field of LLM security!

Best Regards

**Comment by huangtiansheng** [comment]
Thanks for the suggestion. I will update our list after reading your prominent works.  Let's keep doing great research in this field!

Thank you!

**Comment by huangtiansheng** [comment]
Hi,

We have updated our list.  

The high-level idea of  your ML-LR solution is to identify those safety-critical parameters and assign a small learning rate in fine-tuning stage.  We have a post-fine-tuning stage defense Antidote, use a very similar high-level idea. Our idea is to identify those safety-critical parameters and remove them (sparsify to 0) after fine-tuning. Feel free to check out our Antidote (https://arxiv.org/abs/2408.09600) paper if you are interested.

Thanks again,
Tiansheng Huang

**Comment by DYR1** [comment]
Thank you for adding our work to the list! We are glad that we have focused on the same problem and are working on solving it. We will read this paper carefully and hope to have the opportunity to communicate in the future.
